server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/create/CreateSnapshotRequest.java

```diff
@@ -70,7 +70,7 @@
 
     private String[] indices = EMPTY_ARRAY;
 
-    private IndicesOptions indicesOptions = IndicesOptions.strictExpandOpen();
+    private IndicesOptions indicesOptions = IndicesOptions.strictExpand();
```

Comment:
this will be changed back to `IndicesOptions.strictExpandOpen` for the 7.x backport.

Reply:
Thanks very much for your interest in Elasticsearch @vishalk663. 

This appears to be a user question, and we'd like to direct these kinds of things to the [Elasticsearch forum](https://discuss.elastic.co/c/elasticsearch). If you can stop by there, we'd appreciate it. This allows us to use GitHub for verified bug reports, feature requests, and pull requests.

There's an active community in the [forum](https://discuss.elastic.co/c/elasticsearch) that should be able to help get an answer to your question. As such, I hope you don't mind that I mark this message as off-topic.